scripts/setupvars/setupvars.sh

```diff
@@ -72,8 +72,8 @@ if [ -f "$INTEL_OPENVINO_DIR/extras/dl_streamer/setupvars.sh" ]; then
     source "$INTEL_OPENVINO_DIR/extras/dl_streamer/setupvars.sh"
 fi
 
-export PATH="$INTEL_OPENVINO_DIR/tools/model_optimizer${PATH:+:$PATH}"
-export PYTHONPATH="$INTEL_OPENVINO_DIR/tools/model_optimizer${PYTHONPATH:+:$PYTHONPATH}"
+export PATH="$INTEL_OPENVINO_DIR/tools/mo${PATH:+:$PATH}"
+export PYTHONPATH="$INTEL_OPENVINO_DIR/tools/mo${PYTHONPATH:+:$PYTHONPATH}"
```

Comment:
These lines are not relevant any more and should be remove, since we remove mo from installer

Reply:
@slyubimt thanks for noting that! I will remove them

Reply:
removed